### DISCUSSION TITLE
Add new report types

### DISCUSSION THREAD

**Discussion by 282405989** [saleweaver/python-amazon-sp-api#318] [Feature requests]
There's some report API that I found in Amazon-sp-API, but I didn't find those here... can these be added to python-amazon-sp-api?

GET_BRAND_ANALYTICS_MARKET_BASKET_REPORT
GET_BRAND_ANALYTICS_REPEAT_PURCHASE_REPORT
GET_BRAND_ANALYTICS_ALTERNATE_PURCHASE_REPORT
GET_BRAND_ANALYTICS_ITEM_COMPARISON_REPORT
GET_VENDOR_NET_PURE_PRODUCT_MARGIN_REPORT
GET_MERCHANTS_LISTINGS_FYP_REPORT

**Comment by saleweaver** [comment]
Of course, thanks. You can create a pull request with the new types or I'll add them some time during the week.

**Reply by 282405989** [reply]
I've created a pull request, it was my first time to do it. I hope I could be helpful :)

[#](https://github.com/saleweaver/python-amazon-sp-api/pull/319)

**Reply by saleweaver** [reply]
Thank you, it's now merged.
